Formula/dcraw.rb

```diff
@@ -21,12 +21,16 @@ class Dcraw < Formula
   end
 
   depends_on "jasper"
-  depends_on "jpeg"
+  depends_on "jpeg-turbo"
   depends_on "little-cms2"
 
   def install
-    ENV.append_to_cflags "-I#{HOMEBREW_PREFIX}/include -L#{HOMEBREW_PREFIX}/lib"
-    system ENV.cc, "-o", "dcraw", ENV.cflags, "dcraw.c", "-lm", "-ljpeg", "-llcms2", "-ljasper"
+    ENV.append_to_cflags "-I#{Formula["jpeg-turbo"].opt_include}"
+    ENV.append_to_cflags "-I#{HOMEBREW_PREFIX}/include"
+    ENV.append "LDFLAGS", "-L#{Formula["jpeg-turbo"].opt_lib}"
+    ENV.append "LDFLAGS", "-L#{HOMEBREW_PREFIX}/lib"
+    system ENV.cc, "dcraw.c", "-o", "dcraw", *ENV.cflags.to_s.split, *ENV.ldflags.to_s.split,
+                   "-lm", "-ljpeg", "-llcms2", "-ljasper"
```

Comment:
```suggestion
    ENV.append "LDFLAGS", "-L#{HOMEBREW_PREFIX}/lib"
    ENV.append "LDFLAGS", "-lm -ljpeg -llcms2 -ljasper"
    system "make", "dcraw"
```
I think this will work. Might need
```ruby
system "make", "CFLAGS=#{ENV.cflags}", "LDFLAGS=#{ENV.ldflags}", "dcraw"
```
(this works even if upstream do not ship a `Makefile`)

Reply:
Seems to work locally. I do see the superenv removes `-I#{HOMEBREW_PREFIX}/include` so may drop that.
```
clang called with: -I/usr/local/opt/jpeg-turbo/include -I/usr/local/include -L/usr/local/opt/jpeg-turbo/lib -L/usr/local/lib -lm -ljpeg -llcms2 -ljasper dcraw.c -o dcraw
superenv removed:  -I/usr/local/include
superenv added:    -pipe -w -Os -march=nehalem -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk --sysroot=/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk -isystem/usr/local/include -isystem/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -L/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries -Wl,-headerpad_max_install_names
superenv executed: clang -pipe -w -Os -march=nehalem -I/usr/local/opt/jpeg-turbo/include -L/usr/local/opt/jpeg-turbo/lib -lm -ljpeg -llcms2 -ljasper dcraw.c -o dcraw -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk --sysroot=/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk -isystem/usr/local/include -isystem/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -I/usr/local/opt/jpeg-turbo/include -L/usr/local/opt/jpeg-turbo/lib -L/usr/local/lib -L/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries -Wl,-headerpad_max_install_names
```

Reply:
Looks like it breaks on Linux due to argument ordering, e.g. `-lm -ljpeg -llcms2 -ljasper dcraw.c -o dcraw`